### PR TITLE
Fix inconsistencies of types in PointerTensor serialization

### PR DESCRIPTION
**Pull request by Jasopaum**
## Description

Fix inconsistencies of types in PointerTensor serialization.
Also use ptr.id_at_location instead of the whole pointer in CommunicationActions.

Fixes a part of #2887

## Type of change

Please mark the options that are relevant.

- [ ] Added/Modified tutorials
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

* [ ] I have added tests for my changes
* [x] I did follow the [contribution guidelines](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md)
* [ ] I have commented my code following [Google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

### CODE REVIEW

File: syft/workers/base.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def __init__(
         # storage object for crypto primitives
         self.crypto_store = PrimitiveStorage(owner=self)
         # declare the plans used for crypto computations
-        sy.frameworks.torch.mpc.fss.initialize_crypto_plans(self)
+        # sy.frameworks.torch.mpc.fss.initialize_crypto_plans(self)
```
Comment (gmuraru) [inline syft/workers/base.py:185]: Should be uncommented?

Comment (karlhigley) [reply syft/workers/base.py:185]: That actually needs to get fixed. There's a separate issue for it: #3464

Comment (Jasopaum) [reply syft/workers/base.py:185]: Oops, I had commented that only for local debug, thanks for catching that!

File: syft/execution/communication.py [hunk 2]
```diff
@@ -14,7 +14,9 @@
     "mid_get",
     "move",
     "remote_get",
-    "remote_send",
+    "remote_copy_to",
```
Comment (karlhigley) [inline syft/execution/communication.py:17]: I realize I suggested this name, but I might have a better one. What do you think about `remote_send_copy()`?

File: syft/execution/communication.py [hunk 3]
```diff
@@ -14,7 +14,9 @@
...
+    "move_and_point",
```
Comment (karlhigley) [inline syft/execution/communication.py:18]: For consistency, seems like this should maybe be `remote_move_and_point`? 🤔

Comment (Jasopaum) [reply syft/execution/communication.py:18]: I was definitely not found of `move_and_point ` ^^
Do you think we can find a better naming completely?

File: test/serde/serde_helpers.py [hunk 4]
```diff
@@ -1577,7 +1577,7 @@ def make_tensor_command_message(**kwargs):
     cmd2 = bob._get_msg(-1)
 
     x = torch.tensor([1, 2, 3, 4]).send(bob)
-    x.remote_send(alice)
+    x.remote_copy_to(alice)
```
Comment (iamtrask) [inline test/serde/serde_helpers.py:1580]: I really like this functionality - "remote_copy_to" - very nice

File: syft/execution/communication.py (hunk 2 above)
Comment (iamtrask) [reply syft/execution/communication.py:17]: I like remote copy to - not too opinionated

File: syft/execution/communication.py (hunk 3 above)
Comment (iamtrask) [reply syft/execution/communication.py:18]: agreed - remote_move_and_point is better but it's still a little hard to grok. Not a deal breaker though.

**Pull request closed without merging**